pkg/kind/kind.go

```diff
@@ -38,6 +45,50 @@ func SetUp() error {
 }
 
 func createKindCluster() error {
-	fmt.Println("TODO: Creating Kind cluster...")
+	fmt.Println("Creating Kind cluster...")
+
+	// Get kind config file
+	kindConfig, err := ioutil.TempFile(os.TempDir(), "kind-config-*.yaml")
```

Comment:
Jfyi ioutil.TempFile already uses the default tempdir if you leave its first argument as "" (https://golang.org/pkg/io/ioutil/#TempFile), so this could have been:

```suggestion
	kindConfig, err := ioutil.TempFile("", "kind-config-*.yaml")
```